src/class-wpml-elementor-translate-ids.php

```diff
@@ -22,6 +23,30 @@ public function translate_theme_location_template_id( $template_id ) {
 		return $this->translate_id( $template_id );
 	}
 
+	/**
+	 * @param int|string $sub_id
+	 * @param array      $parsed_condition
+	 *
+	 * @return int|string
+	 */
+	public function translate_location_condition_sub_id( $sub_id, $parsed_condition ) {
+		$sub_name = isset( $parsed_condition['sub_name'] ) ? $parsed_condition['sub_name'] : null;
+
+		if ( (int) $sub_id > 0 && $sub_name ) {
+			$element_type = $sub_name;
+
+			if ( 'child_of' === $sub_name ) {
```

Comment:
@strategio It might be a good idea to add a short comment about what `$sub_name` means.